src/CmsChangeLogBundle/src/esono/pkgcmschangelog/TCMSTableEditor/TCMSTableEditorChangeLog.class.php

```diff
@@ -62,13 +63,20 @@ public function Save(&$postData, $bDataIsInSQLForm = false)
     {
         if ($this->oTableConf->fieldChangelogActive) {
             $this->failOnForbiddenTables();
-            $this->savePreSaveValues($postData);
+
+            if (isset($postData['id'])) {
```

Comment:
This can be simplified --> `$this->bIsUpdate = isset($postData['id']);`

(also in SaveField())

Reply:
ok